client/components/edit/datastream/DatastreamControlButton.tsx

```diff
@@ -15,14 +24,17 @@ const DatastreamControlButton = ({ modalState, datastream }) => {
     };
 
     return (
-        <Button onClick={onClick} variant="outlined" size="small">
-            {modalState}
-        </Button>
+        <Tooltip title={modalState}>
+            <IconButton disabled={modalState !== "Upload" && disabled} onClick={onClick} size="small">
+                {Icons[modalState]}
```

Comment:
Should we put some alt-text or tool tips on these?

Reply:
Added aria-label for alternative text for <svg> icons.